### PR TITLE
Clean up PEP-505.

### PR DESCRIPTION
**Pull request by mehaase**
- Replace emoji with C# spelling for operators.
- Remove some unnecessary alternative spellings.
- Add proper python code blocks.
- Other miscellany.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

Unfortunately our records indicate you have not signed a [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA). For legal reasons we need you to sign this before we can look at your contribution.

Once you have done everything that's needed, please reply here and someone will verify everything is in order.

**Comment by mehaase** [comment]
Signed contributor agreement has been submitted.

**Comment by brettcannon** [comment]
The build check failed, @mehaase, because pygments isn't installed for the PEPs. I think if you remove your code block directives -- e.g. `.. code:: csharp` -- that will solve the build failure.

File: pep-0505.txt [hunk 1]
```diff
@@ -302,7 +315,7 @@ from a SQL database and formats it as JSON to send to an HTTP client::
 
 Both ``first_seen`` and ``last_seen`` are allowed to be ``null`` in the
 database, and they are also allowed to be ``null`` in the JSON response. JSON
-does not have a native way to represent a ``datetime``, so the server's
+does not have a native way to represent a ``datetime``, so the the server's
```
Comment (Mariatta) [inline pep-0505.txt]: You have an extra `the` here.

File: pep-0505.txt [hunk 2]
```diff
@@ -504,10 +526,12 @@ Usage Of ``None`` In The Standard Library
 
 The previous sections show some code patterns that are claimed to be "common",
 but how common are they? The attached script `find-pep505.py
-<https://github.com/python/peps/blob/master/pep-0505/find-pep505.py>`_ is meant to
+<https://hg.python.org/peps/file/tip/pep-0505/find-pep505.py>`_ is meant to
```
Comment (Mariatta) [inline pep-0505.txt]: Is there a reason for pointing to mercurial peps repo instead of github?

File: pep-0505.txt [hunk 3]
```diff
@@ -577,11 +603,11 @@ The script prints out any matches it finds. Sample::
     hand, we assume that ``and`` is always incorrect for safe navigation.
 
 The script has been tested against `test.py
-<https://github.com/python/peps/blob/master/pep-0505/test.py>`_ and the Python 3.4
+<https://hg.python.org/peps/file/tip/pep-0505/test.py>`_ and the Python 3.4
```
Comment (Mariatta) [inline pep-0505.txt]: Use github url here?

File: pep-0505.txt [hunk 4]
```diff
@@ -577,11 +603,11 @@ The script prints out any matches it finds. Sample::
...
 standard library, but it should work on any arbitrary Python 3 source code. The
 complete output from running it against the standard library is attached to this
-proposal as
-`find-pep505.out <https://github.com/python/peps/blob/master/pep-0505/find-pep505.out>`_.
+proposal as `find-pep505.out <https://hg.python.org/peps/raw-file/tip/pep-0505
```
Comment (Mariatta) [inline pep-0505.txt]: here too

File: pep-0505.txt (hunk 2 above)
Comment (mehaase) [inline pep-0505.txt]: Sorry, I didn't realize that somebody rewrote this link when I submitted it to the peps e-mail list last year. Fixed in latest commit.

**Comment by gvanrossum** [comment]
I've merged this for you. That does not mean an endorsement. If you have further updates please create a new pull request.

**Pull request merged**